Core/Core/Transports/ServerUtils/ServerAPI.cs

```diff
@@ -18,16 +18,21 @@ namespace Speckle.Core.Transports.ServerUtils;
 
 public sealed class ServerApi : IDisposable, IServerApi
 {
-  private const int BatchSizeGetObjects = 10000;
-  private const int BatchSizeHasObjects = 100000;
+  private const int BATCH_SIZE_GET_OBJECTS = 10000;
+  private const int BATCH_SIZE_HAS_OBJECTS = 100000;
+
+  private const int MAX_MULTIPART_COUNT = 5;
+  private const int MAX_MULTIPART_SIZE = 25_000_000;
+  private const int MAX_OBJECT_SIZE = 25_000_000;
+
+  private const int MAX_REQUEST_SIZE = 100_000_000;
+
+  private const int RETRY_COUNT = 3;
+  private static readonly HashSet<int> s_retryCodes = new() { 408, 502, 503, 504 };
+  private static readonly char[] s_separator = { '\t' };
+  private static readonly string[] s_filenameSeparator = { "filename=" };
```

Comment:
We should consider moving these to a "magic numbers" class at some point, as ideally all these values should be defined by the server.